### PR TITLE
feat(create-contentful-studio-experiences): rename package for the CLI tool [ALT-991]

### PR DESCRIPTION
**Pull request by primeinteger**
## Purpose

* Renames the CLI tool package `create-experience-builder` -> `create-contentful-studio-experiences`
  * This changes the command to run the CLI tool to `npm init contentful-studio-experiences` or `npx contentful-studio-experiences`
* Updates "Experience Builder" verbiage to "Studio Experiences" in the CLI tool.

### Other changes

* Readme updates for the CLI tool (mostly small formatting changes) - [View rendered version](https://github.com/contentful/experience-builder/tree/ALT-991/cli-updates/packages/create-contentful-studio-experiences#readme)
* Separates naming for the Experience Type and Demo Experience Entry:
  * Previously, the script was blending the use of "Experience Name" as both the Experience Type and Demo Experience Entry.
  * Now the Experience Type is created as "Studio Experiences" (default value, user can customize) and the Demo Experience Entry is created as "Home Page" (to match the slug in the template app).
* Small styling changes for the Demo Experience Entry:
  * Arial font added to App.css to avoid showing Times New Roman

### Artwork

Screenshot 2024-06-24 at 2 46 47 PM

Screenshot 2024-06-24 at 2 47 52 PM

Screenshot 2024-06-24 at 2 48 04 PM

Screenshot 2024-06-24 at 2 47 24 PM

### CODE REVIEW

File: packages/create-contentful-studio-experiences/src/index.ts [hunk 1]
```diff
@@ -207,72 +211,54 @@ async function init() {
...
+    const previewEnvironments = await ctflClient.getPreviewEnvironments();
 
-    await ctflClient.createContentEntry(experienceName, experienceNameId);
+    const previewEnv = previewEnvironments.find(
+      (item) => item.url === `http://localhost:${variant.devPort}`,
+    );
+
+    if (previewEnvironments.length === 0 || !previewEnv) {
+      spinner.start(
+        'No preview environments for Studio Experiences currently found, creating one.',
```
Comment (elylucas) [inline packages/create-contentful-studio-experiences/src/index.ts:240]:
```suggestion
        'No content preview environments for Studio Experiences currently found, creating one.',
```

File: packages/create-contentful-studio-experiences/src/index.ts [hunk 2]
```diff
@@ -207,72 +211,54 @@ async function init() {
...
+      );
+      await ctflClient.createPreviewEnvironment(variant.devPort, contentTypeName, contentTypeId);
+      spinner.stop('Preview environment created!');
+    } else if (previewEnv) {
+      log.message('Using existing preview environment for Studio Experiences.');
```
Comment (elylucas) [inline packages/create-contentful-studio-experiences/src/index.ts:245]:
```suggestion
      log.message('Using existing content preview environment for Studio Experiences.');
```

File: packages/create-contentful-studio-experiences/src/index.ts [hunk 3]
```diff
@@ -207,72 +211,54 @@ async function init() {
...
+    const previewEnv = previewEnvironments.find(
+      (item) => item.url === `http://localhost:${variant.devPort}`,
+    );
+
+    if (previewEnvironments.length === 0 || !previewEnv) {
+      spinner.start(
+        'No preview environments for Studio Experiences currently found, creating one.',
+      );
+      await ctflClient.createPreviewEnvironment(variant.devPort, contentTypeName, contentTypeId);
+      spinner.stop('Preview environment created!');
```
Comment (elylucas) [inline packages/create-contentful-studio-experiences/src/index.ts:243]:
```suggestion
      spinner.stop('Content preview environment created!');
```

File: packages/create-contentful-studio-experiences/package.json [hunk 4]
```diff
@@ -1,9 +1,8 @@
 {
-  "name": "@contentful/create-experience-builder",
-  "private": true,
```
Comment (elylucas) [inline packages/create-contentful-studio-experiences/package.json:3]: we might want to keep this private on to keep the package from publishing until its ready to go

**Review by elylucas — APPROVED** [review]
Overall looks great. I left some suggestions to be more specific about "content preview entries" vs just "preview entries", but nothing to not hold up an approval.

File: packages/create-contentful-studio-experiences/package.json (hunk 4 above)
Comment (primeinteger) [reply packages/create-contentful-studio-experiences/package.json:3]: Ok sounds good, I'll set it back to private in the next commit

**Pull request merged**